client/verta/tests/test_config.py

```diff
@@ -147,6 +150,12 @@ def test_merge_and_overwrite(self, expected_config):
         with _config_utils.read_merged_config() as config:
             assert config == expected_config
 
+    def test_skip_client_load_config(self, expected_config):
+        assert Client()._config == expected_config
+
+        os.environ["VERTA_DISABLE_CLIENT_CONFIG"] = "1"
```

Comment:
This sets the flag forever and for the next tests. Not sure if it will impact something

Reply:
Whoops thanks!

Reply:
`pytest` is gosh-darn magic

https://docs.pytest.org/en/stable/how-to/monkeypatch.html#monkeypatching-environment-variables